Tests/SwiftDocCTests/Test Bundles/MarkdownOutput.docc/MarkdownOutput.symbols.json

```diff
@@ -0,0 +1 @@
berOf","source":"s:14MarkdownOutput0A6SymbolV4nameSSvp","target":"s:14MarkdownOutput0A6SymbolV"},{"kind":"memberOf","source":"s:14MarkdownOutput0A6SymbolV8fullNameSSvp","target":"s:14MarkdownOutput0A6SymbolV"},{"kind":"memberOf","source":"s:14MarkdownOutput0A6SymbolV9otherNameSSSgvp","target":"s:14MarkdownOutput0A6SymbolV"},{"kind":"memberOf","source":"s:14MarkdownOutput0A6SymbolV4nameACSS_tcfc","target":"s:14MarkdownOutput0A6SymbolV"},{"kind":"memberOf","source":"s:SQsE2neoiySbx_xtFZ::SYNTHESIZED::s:14MarkdownOutput17ExternalConformerV","target":"s:14MarkdownOutput17ExternalConformerV","sourceOrigin":{"identifier":"s:SQsE2neoiySbx_xtFZ","displayName":"Equatable.!=(_:_:)"}},{"kind":"conformsTo","source":"s:14MarkdownOutput17ExternalConformerV","target":"s:Se","targetFallback":"Swift.Decodable"},{"kind":"conformsTo","source":"s:14MarkdownOutput17ExternalConformerV","target":"s:SE","targetFallback":"Swift.Encodable"},{"kind":"conformsTo","source":"s:14MarkdownOutput17ExternalConformerV","target":"s:s12IdentifiableP","targetFallback":"Swift.Identifiable"},{"kind":"conformsTo","source":"s:14MarkdownOutput17ExternalConformerV","target":"s:SH","targetFallback":"Swift.Hashable"},{"kind":"conformsTo","source":"s:14MarkdownOutput17ExternalConformerV","target":"s:SQ","targetFallback":"Swift.Equatable"},{"kind":"memberOf","source":"s:14MarkdownOutput17ExternalConformerV2idSSvp","target":"s:14MarkdownOutput17ExternalConformerV","sourceOrigin":{"identifier":"s:s12IdentifiableP2id2IDQzvp","displayName":"Identifiable.id"}},{"kind":"memberOf","source":"s:14MarkdownOutput17ExternalConformerV4fromACs7Decoder_p_tKcfc","target":"s:14MarkdownOutput17ExternalConformerV","sourceOrigin":{"identifier":"s:Se4fromxs7Decoder_p_tKcfc","displayName":"Decodable.init(from:)"}},{"kind":"memberOf","source":"s:SQsE2neoiySbx_xtFZ::SYNTHESIZED::s:14MarkdownOutput14LocalConformerO","target":"s:14MarkdownOutput14LocalConformerO","sourceOrigin":{"identifier":"s:SQsE2neoiySbx_xtFZ","displayName":"Equatable.!=(_:_:)"}},{"kind":"conformsTo","source":"s:14MarkdownOutput14LocalConformerO","target":"s:SQ","targetFallback":"Swift.Equatable"},{"kind":"conformsTo","source":"s:14MarkdownOutput14LocalConformerO","target":"s:SH","targetFallback":"Swift.Hashable"},{"kind":"conformsTo","source":"s:14MarkdownOutput14LocalConformerO","target":"s:14MarkdownOutput13LocalProtocolP"},{"kind":"memberOf","source":"s:14MarkdownOutput14LocalConformerO11localMethodyyF","target":"s:14MarkdownOutput14LocalConformerO","sourceOrigin":{"identifier":"s:14MarkdownOutput13LocalProtocolP11localMethodyyF","displayName":"LocalProtocol.localMethod()"}},{"kind":"memberOf","source":"s:14MarkdownOutput14LocalConformerO3booyA2CmF","target":"s:14MarkdownOutput14LocalConformerO"},{"kind":"requirementOf","source":"s:14MarkdownOutput13LocalProtocolP11localMethodyyF","target":"s:14MarkdownOutput13LocalProtocolP"},{"kind":"inheritsFrom","source":"s:14MarkdownOutput13LocalSubclassC","target":"s:14MarkdownOutput15LocalSuperclassC"}]}
```

Comment:
FYI: You have your `file:///Users/richard/Documents/...` path components in this file. 

If there's some symbol graph data that's really complex and hard to recreate, so that we opt to use an exported file for it, we rename that to `file:///Users/username/path/to/...`